### DISCUSSION TITLE
如何在评论中出现的链接的属性里加入nofollow？

### DISCUSSION THREAD

**Discussion by yzl3014** [walinejs/waline#1863] [Q&A] [answered]
如果评论中涉及到了部分有问题的网站或者广告，而又没空管理，可能会影响到搜索引擎收录。并且，部分重视SEO的站长，也会尽可能的减少不必要的链接，保持权重。
似乎只需要修改一下输出即可，但是我并不了解如何配置。文档没有提及，必应搜索后也没有相关结果。
![评论链接信息](https://s2.loli.net/2023/03/16/TLxiXcgONerAVoa.jpg)

**Comment by lizheming** [comment]
@Mister-Hope Hopes 老师看看 markdown-it 这块有什么方式能快速的支持这个吗？我对 markdown-it 不是特别熟

**Comment by Mister-Hope** [comment]
你的意思是加一个nofollow给所有链接是吧？

**Reply by lizheming** [reply]
是的

**Comment by Mister-Hope** [comment, marked as answer]
Done in https://github.com/walinejs/waline/commit/b4c61653c4f08cbd05228b5d22c512c24b099f60
